src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java

```diff
@@ -205,22 +205,56 @@ public void testGenerateFeedbackSessionEmails() throws IOException {
         
         ______TS("feedback session submission email");
 
-        Calendar time = Calendar.getInstance();
-        time.set(Calendar.DATE, 4);
-        time.set(Calendar.MONTH, 8);
-        time.set(Calendar.HOUR_OF_DAY, 5);
-        time.set(Calendar.MINUTE, 30);
+        Calendar timeForStudent = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        timeForStudent.set(Calendar.DATE, 4);
+        timeForStudent.set(Calendar.MONTH, 8);
+        timeForStudent.set(Calendar.HOUR_OF_DAY, 5);
+        timeForStudent.set(Calendar.MINUTE, 30);
         EmailWrapper email = new EmailGenerator().generateFeedbackSubmissionConfirmationEmailForStudent(
-                session, student1, TimeHelper.formatTime12H(time.getTime()));
+                session, student1, timeForStudent);
         subject = String.format(EmailType.FEEDBACK_SUBMISSION_CONFIRMATION.getSubject(), course.getName(),
                                 session.getFeedbackSessionName());
         verifyEmail(email, student1.email, subject, "/sessionSubmissionConfirmationEmailForStudent.html");
 
+        Calendar timeForInstructor = Calendar.getInstance(TimeZone.getTimeZone("UTC"));
+        timeForInstructor.set(Calendar.DATE, 4);
+        timeForInstructor.set(Calendar.MONTH, 8);
+        timeForInstructor.set(Calendar.HOUR_OF_DAY, 5);
+        timeForInstructor.set(Calendar.MINUTE, 30);
         email = new EmailGenerator().generateFeedbackSubmissionConfirmationEmailForInstructor(session,
-                instructor1, TimeHelper.formatTime12H(time.getTime()));
+                instructor1, timeForInstructor);
         subject = String.format(EmailType.FEEDBACK_SUBMISSION_CONFIRMATION.getSubject(), course.getName(),
                                 session.getFeedbackSessionName());
         verifyEmail(email, instructor1.email, subject, "/sessionSubmissionConfirmationEmailForInstructor.html");
+        
+        ______TS("feedback session submission email with Singapore timezone");
+
+        session.setTimeZone(12.75);
+        Calendar timeForStudentSingapore = Calendar.getInstance(TimeZone.getTimeZone("GMT+08:00"));
```

Comment:
Although this would have been removed due to one of the previous comments, using "GMT+08:00" is not recommended because the `TimeHelper` utility is pre-set to UTC.